lwallet/Command.hs

```diff
@@ -156,6 +163,33 @@ proposeUpdate =
     lexeme parseSoftwareVersion <*>
     many1 parseProposeUpdateSystem
 
+stakeholderIdP :: Parser StakeholderId
+stakeholderIdP = hash
```

Comment:
`stakeholderIdParser` exists in `Pos.Client.CLI.Util`.
I don't like the idea of having multiple ways to parse it. Even if you don't like that definition, let's agree on one of them somehow.

Reply:
The definition there won't work in my case because it will consume the colon which I use as a separator.